common/prque/sstack.go

```diff
@@ -81,7 +115,38 @@ func (s *sstack) Len() int {
 // Compares the priority of two elements of the stack (higher is first).
 // Required by sort.Interface.
 func (s *sstack) Less(i, j int) bool {
-	return (s.blocks[i/blockSize][i%blockSize].priority - s.blocks[j/blockSize][j%blockSize].priority) > 0
+	var result bool
+
+	switch s.priorityType {
+	case Int:
+		i := (s.blocks[i/blockSize][i%blockSize].priority).(int)
+		j := (s.blocks[j/blockSize][j%blockSize].priority).(int)
+		result = i > j
+	case Int64:
+		i := (s.blocks[i/blockSize][i%blockSize].priority).(int64)
+		j := (s.blocks[j/blockSize][j%blockSize].priority).(int64)
+		if i == j {
+			return false
+		}
```

Comment:
Why do we need these lines?

Reply:
It is reflected in https://github.com/klaytn/klaytn/pull/840 PTAL :)